code/modules/admin/verbs/debug.dm

```diff
@@ -948,6 +948,51 @@
 /proc/cmp_timer_data(list/a, list/b)
 	return b["count"] - a["count"]
 
+/client/proc/create_mob_worm()
+	set category = "Debug"
+	set name = "Create Mob Worm"
+	set desc = "Attached a linked list of mobs to a marked mob"
+	if (!check_rights(R_DEBUG))
+		return
+	if(isnull(holder))
+		return
+	if(!isliving(holder.marked_datum))
+		to_chat(usr, span_warning("Error: Please mark a mob to attach mobs to."))
+		return
+	var/mob/living/head = holder.marked_datum
+
+	var/attempted_target_path = input(
```

Comment:
making worms out of people isn't an admin-critical tool (sadly), make this `tgui_input` instead. you use `tgui_input_number` later on so lets be consistent